doc/coadd1d.rst

```diff
@@ -303,3 +303,45 @@ and launches a GUI from the `linetools`_ package. e.g.:
 
     lt_xspec J1217p3905_coadd.fits
 
+UVES_popler coaddition
+======================
+
+If you prefer to use a GUI for the coaddition (to manually remove
+bad pixels, ghosts, cosmic rays etc.), then you can use the UVES_popler tool.
+This tool is developed by Michael Murphy and is available at
+`this link <https://github.com/MTMurphy77/UVES_popler/>`__.
+Here is an example of a coadded spectrum using UVES_popler:
+
+.. image:: ../figures/uves_popler.png
+   :scale: 60%
+
+UVES_popler was originally written to coadd ESO/UVES echelle spectra
+that were reduced by the ESO pipeline, and it has been recently modified
+to support the reduction of PypeIt longslit and echelle data. If you
+require some support on how to use UVES_popler with PypeIt data, please
+join the `PypeIt Users Slack <https://pypeit-users.slack.com>`__ and
+ask for help in the #coadd1d channel. All are welcome to join using
+`this invitation link <https://join.slack.com/t/pypeit-users/shared_invite/zt-1kc4rxhsj-vKU1JnUA~8PZE~tPlu~aTg>`__.
```

Comment:
Thanks.  To be clear, I'm fine with there being a #uves_popler channel in the pypeit users slack where users can discuss that tool specifically.  I just want to keep it distinct from #coadd1d .

Reply:
Totally fine... but I think it's perhaps best that we don't explicitly list that they should receive help from the PypeIt Slack channel. If there's enough comments about it, then we can always add a #uver_popler channel in the Slack, but maybe best not to advertise it as such, since none of us are really able to address issues within UVES_popler. But if there's demand, I'm more than happy to add that in later, provided that Michael Murphy is happy to be the one who provides support

Reply:
Sounds good to me!